README.md

```diff
@@ -91,13 +91,22 @@ See the recipes directory for example usage of cisco providers and resources.
 
 ### <a name="provider-platform-support">Provider Support Across Platforms</a>
 
-* ✅ = Supported
-* ❌ = Unsupported
+A note about support for specific platform models:
 
-|   | N9k | N30xx | N31xx | N56xx | N6k | N7k | Caveats |
+  * "**N9k**" support includes all N9xxx models.
```

Comment:
I suggest we replace this info with the same table that we have in the Puppet README.

https://github.com/cisco/cisco-network-puppet-module#resource-platform-support-matrix

This has a bit more information that is helpful for users.